### PR TITLE
add session_type api to testing

### LINKED ISSUE #3928: Add session types api
parent issue: #3870

### PR DESCRIPTION
**Pull request by shubham-padia**
fixes #3928

### CODE REVIEW

File: app/api/session_types.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def validate_length(self, data):
     id = fields.Str(dump_only=True)
     name = fields.Str(required=True)
     length = fields.Str(required=True)
-    event = Relationship(attribute='event',
+    event = Relationship(attribute='events',
```
Comment (niranjan94) [inline app/api/session_types.py]: Why is the attribute called as `events`? Shouldn't it be singular?

Comment (shubham-padia) [inline app/api/session_types.py]: Resolved @niranjan94

**Comment by shubham-padia** [comment]
@niranjan94 please review

**Comment by shubham-padia** [comment]
@niranjan94 please review

**Comment by niranjan94** [comment]
@shubham-padia 
looks good. Though you need to resolve conflicts here.

**Comment by shubham-padia** [comment]
Oops !! missed that. :sweat_smile: 

@enigmaeth @niranjan94 Can you please review again

**Pull request merged**